ytmusicapi/__init__.py

```diff
@@ -1,7 +1,7 @@
 from importlib.metadata import PackageNotFoundError, version
 
-from ytmusicapi.setup import setup, setup_oauth
```

Comment:
Please undo the import changes and try to keep the changeset to the feature request. In general, it is a good idea to keep changesets minimal to ensure smooth sailing and quick merges

Reply:
Not sure what you meant by "get proper module linking to track down the required functions".

If you configure `ytmusicapi` as the module/source folder in your IDE the function navigation should work without issue, at least it does for me in Pycharm

Reply:
sure thing.